### ISSUE TITLE
Missing successful global message when user sets additional address as default

### ISSUE THREAD

**Issue by developpeurweb** [SAP/spartacus#13719]
**Describe the bug**
- Screen reader users wouldn't be able to tell if they have changed an address as the new default.

**Tell us the version of Spartacus**
- Library version: 4.0.0

**To Reproduce**
1. Go to My Account > Address Book
2. Add 2 different addresses or more.
3. Open any Screen Reader (VO, NVDA, JAWS).
4. Switch addresses from additional to new defaults.
5. Listen to Screen Reader feedback
6. Hear the absence of confirmation a change has happened.

**Expected behavior**
- Screen reader users will hear a successful confirmation message as they set an address as new default. E.g _"Additional address successfully set as new default"._
- For `MSG_TYPE_CONFIRMATION` use `aria-live="polite" aria-atomic="true"`

**Screenshots**

Message suggestion in the screenshot below, please validate phrasing with @Manurod2018 or @vekterdude

![a2021-09-14_20-51-14](https://user-images.githubusercontent.com/15113219/133353447-c25ff4d3-fd2f-45af-955b-6c1eedf62bce.jpg)

**Comment by KateChuen** [comment]
Tested on develop branch of spartacus using installation script
More extensive testing (on other screenreaders and devices) will be done with RC.
![AddDefault](https://user-images.githubusercontent.com/45409515/149587523-e4b7352a-738e-4edf-a2f5-240123bd7fbb.png)
